packages/webapp/src/components/DateRangeSelector/CustomDateRangeSelector.jsx

```diff
@@ -0,0 +1,81 @@
+/*
+ *  Copyright 2023 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+import clsx from 'clsx';
+import { BsChevronLeft } from 'react-icons/bs';
+import { Semibold, Underlined } from '../Typography';
+import { FromToDateContainer } from '../Inputs/DateContainer';
+import styles from './styles.module.scss';
+
+export default function CustomDateRangeSelector({
+  onBack,
+  onClear,
+  isValid,
+  validRange,
+  startDate,
+  endDate,
+  changeStartDate,
+  changeEndDate,
+  fromDateMax,
+  toDateMin,
+}) {
+  const { t } = useTranslation();
+  const inputStyle = { classes: { container: { minWidth: 'calc((100% - 24px) / 2)' } } };
+
+  return (
+    <div className={styles.customDateRangeSelector}>
+      <div className={styles.buttons}>
+        <div
+          className={clsx(styles.backButton, !isValid && styles.disabled)}
+          onClick={isValid ? onBack : undefined}
+        >
+          <BsChevronLeft />
+          {t('DATE_RANGE_SELECTOR.BACK')}
+        </div>
+        <Underlined className={styles.clearDates} onClick={onClear}>
+          {t('DATE_RANGE_SELECTOR.CLEAR_DATES')}
+        </Underlined>
+      </div>
+      <FromToDateContainer
+        onStartDateChange={changeStartDate}
+        onEndDateChange={changeEndDate}
+        endDate={endDate}
+        startDate={startDate}
+        fromProps={{ max: fromDateMax, ...inputStyle }}
+        toProps={{ min: toDateMin, ...inputStyle }}
+      />
+      {startDate && endDate && !validRange && (
+        <Semibold style={{ textAlign: 'center', color: 'red' }}>
```

Comment:
I copied the existing code. I don't like `color: 'red'` but will leave it for now.